cpu/BUILD

```diff
@@ -153,12 +153,26 @@ constraint_value(
     constraint_setting = ":cpu",
 )
 
+# Base integer instruction set, 32-bit.
 constraint_value(
     name = "riscv32",
     constraint_setting = ":cpu",
 )
 
+# Base integer instruction set, 32-bit, 16 registers.
+constraint_value(
+    name = "riscv32e",
+    constraint_setting = ":cpu",
+)
+
+# Base integer instruction set, 64-bit.
 constraint_value(
     name = "riscv64",
     constraint_setting = ":cpu",
 )
+
+# Base integer instruction set, 64-bit, 16 registers.
```

Comment:
Is there any opinion from the riscv expert about what the (inevitable) 64-bit 32-register core will be called? Or, to put it backwards, is the 'e' an official name or is it just what we have been using internally?